test/executor/update_test.cpp

```diff
@@ -152,21 +152,25 @@ TEST_F(UpdateTests, UpdatingOld) {
   catalog->CreateDatabase(DEFAULT_DB_NAME, txn);
   LOG_INFO("Bootstrapping completed!");
 
-  optimizer::Optimizer optimizer;
+  std::unique_ptr<optimizer::AbstractOptimizer> optimizer;
+  optimizer.reset(new optimizer::Optimizer);
+
+
   auto& traffic_cop = tcop::TrafficCop::GetInstance();
 
+
   // Create a table first
   LOG_INFO("Creating a table...");
-  auto id_column = catalog::Column(type::TypeId::INTEGER,
-                                   type::Type::GetTypeSize(type::TypeId::INTEGER),
+  auto id_column = catalog::Column(type::INTEGER,
+                                   type::Type::GetTypeSize(type::INTEGER),
```

Comment:
This seems incorrect. All of the code uses `type::TypeId::INTEGER` but you have `type::INTEGER`. Why did you change this?